src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -71,30 +88,29 @@ function RetroCertsCertificationPage(props) {
                 </Button>
               </div>
               <div className="col-md-4">
-                <Button
-                  variant="secondary"
-                  as={Link}
-                  to={
-                    weekForUser === numberOfWeeks
-                      ? routes.retroCertsConfirmation
-                      : routes.retroCertsCertify +
-                        "/" +
-                        fromIndexToPathString(
-                          userData.weeksToCertify[weekForUser]
-                        )
-                  }
-                  type="submit"
-                >
-                  {weekForUser === numberOfWeeks ? (
-                    t("retrocerts-certification.button-submit")
-                  ) : (
+                {weekForUser === numberOfWeeks && (
```

Comment:
no need to change, just curious, is this functionally different or better than using `(weekForUser === numberOfWeeks) ? (Button) : (otherButton)`?

Reply:
I think it's more readable? But yeah, there's no functional difference.